### DISCUSSION TITLE
How to change margins' size?

### DISCUSSION THREAD

**Discussion by rafaelafguerra** [joaomlourenco/novathesis#408] [Q&A] [answered]
Hello,
I have tried multiple approaches and I am not able to change the size of the margins (for all of the chapters).
How can I do this?
Thank you.

Kind regards,
Rafaela Guerra

**Comment by joaomlourenco** [comment, marked as answer]
DISCLAIMER: Please notice that each University/School has its own (strict) regulations on the page layout.  If you mess up with it, you rest it won't be accepted by the services.

In file `Config/5_packages.tex` add the following (and adapt the margins to the values you want):
```latex
% Set margins for "normal" pages
\margin(screen,top):={3.5cm}
\margin(screen,bottom):={2.5cm}
\margin(screen,left):={3.0cm}
\margin(screen,right):={3.0cm}

\margin(paper,top):={3.5cm}
\margin(paper,bottom):={2.5cm}
\margin(paper,left):={3.0cm}
\margin(paper,right):={3.0cm}
```

**Reply by rafaelafguerra** [reply]
Thank you, I did not know about those strict regulations, in that case, I will stick to the layout, but thank you anyway.
